name: "Jetracer"
N: 8
integrator_step: 0.3
n_discs: 0
track_choice: "t_junction" # 'straight_line', sinus, "t_junction"

number_of_robots: 2
robot_1:
  start_x: -5.0
  start_y: 0.0
  start_theta: 0.0  # in radians
robot_2:
  start_x: 1.0
  start_y: -3.0
  start_theta: 0.5  # radians

verbose: false
debug_visuals: false
dart_simulator: true

solver_settings:
  iterations: 1 #2 # 1
  solver_type: SQP_RTI # SQP_RTI (default) or SQP

control_frequency: 20
braking_acceleration: 1.0

polytopic:
  d_min: 0.2
  length: 0.65
  width: 0.65
  com_to_back: 0.0

contouring:
  num_segments: 5
  use_huber: false

weights:
  goal: 2.
  throttle: .1
  steering: .1
  velocity: 0.5
  reference_velocity: 1.
  lag: 0.2
  contour: 0.2
  terminal_angle: 5.0 #0.0
  terminal_contour: 1.0 # 0.0
  terminal_lag: 1.0 # 0.0






